src/test/ui/coherence/impl[t]-foreign[fundamental[t]]-for-foreign.stderr

```diff
@@ -1,17 +1,17 @@
 error[E0210]: type parameter `T` must be used as the type parameter for some local type (e.g., `MyStruct<T>`)
-  --> $DIR/impl[t]-foreign[fundamental[t]]-for-foreign.rs:10:6
+  --> $DIR/impl[t]-foreign[fundamental[t]]-for-foreign.rs:10:17
    |
 LL | impl<T> Remote1<Box<T>> for u32 {
-   |      ^ type parameter `T` must be used as the type parameter for some local type
+   |                 ^^^^^^ type parameter `T` must be used as the type parameter for some local type
```

Comment:
@lcnr This is one of the last (visual) kinks... I was able to detect the fundamental type for cases like `impl<T> Remote for Box<T>` where the `T` is at idx=0, and I can look at `self_ty` which is a rustc_middle::Ty.

But in this case, I only have a `hir::Ty` and can't figure out how it's a fundamental type from that.

Reply:
urgh the link broke after rebase+force push, https://github.com/rust-lang/rust/pull/100555/files#diff-fddbd2e2a2a86430f2f013d5d787d7ad6c85427cd81a2282ac0fcc7d641733bc